src/field_theory/finite.lean

```diff
@@ -258,3 +266,17 @@ begin
   simpa only [-zmod.pow_totient, nat.succ_eq_add_one, nat.cast_pow, units.coe_one,
     nat.cast_one, cast_unit_of_coprime, units.coe_pow],
 end
+
+/-- A variation on Fermat's little theorem. See `zmod.fermat_little` -/
+@[simp] lemma zmod.pow_card {p : ℕ} [fact p.prime] (x : zmod p) : x ^ p = x :=
```

Comment:
I wonder if perhaps both this lemma and `zmod.fermat_little` belong somewhere where they are more likely to be found.

Reply:
Also, feel free to rename `fermat_little` to something that follows the naming conventions.

Reply:
Where do you think they're likely to be found? I feel like they both belong in `field_theory/finite`, unless there should be a file in `data/zmod` that imports it, but that doesn't sound easier to find.

Reply:
I've put everything at the end of `field_theory/finite`. Does this seem reasonable?